ts/features/bonus/cgn/components/merchants/CgnMerchantsDiscountItem.tsx

```diff
@@ -45,19 +45,20 @@ const CgnMerchantDiscountItem: React.FunctionComponent<Props> = ({
               />
               <View hspacer small />
               <View style={IOStyles.flex}>
+                {/* FIXME properly handle multiple categories on discount */}
                 <H5 weight={"SemiBold"} color={"bluegrey"}>
-                  {discount.category.toLocaleUpperCase()}
+                  {discount.productCategories[0].toLocaleUpperCase()}
```

Comment:
```suggestion
                 {index(0, [...discount.productCategories]).fold(
                undefined,
                cat => (
                  <View
                    style={IOStyles.flex}
                    key={`merchant_category_${discount.discount}`}
                  >
                    {/* FIXME properly handle multiple categories on discount */}
                    <H5 weight={"SemiBold"} color={"bluegrey"}>
                      {cat.toLocaleUpperCase()}
                    </H5>
                  </View>
                )
              )}
```
I suggest to be safe as possibile.
If `discount.productCategories` has 0 items, this code produces a crash